Gem/Code/Source/Components/NetworkMatchComponent.cpp

```diff
@@ -6,9 +6,31 @@
  */
 
 #include <Source/Components/NetworkMatchComponent.h>
+#include <UiGameOverBus.h>
 
 namespace MultiplayerSample
 {
+    void NetworkMatchComponent::Reflect(AZ::ReflectContext* context)
+    {
+        AZ::SerializeContext* serializeContext = azrtti_cast<AZ::SerializeContext*>(context);
+        if (serializeContext)
+        {
+            serializeContext->Class<NetworkMatchComponent, NetworkMatchComponentBase>()
+                ->Version(1);
+        }
+        NetworkMatchComponentBase::Reflect(context);
+    }
+
+    void NetworkMatchComponent::HandleRPC_EndMatch(
+        [[maybe_unused]] AzNetworking::IConnection* invokingConnection, [[maybe_unused]] const MatchResultsSummary& results)
+    {
+        AZ_TracePrintf("NetworkMatchComponent", "Handling RPC_EndMatch!\n", "...");
+        UiGameOverBus::Broadcast(&UiGameOverBus::Events::SetGameOverScreenEnabled, true);
```

Comment:
It sounds like you can combine these calls but up to you.

Reply:
wanted to allow for separate triggering in the future in case there was a delay between wanting to end the game and show fully populated results